SwiftPMTests/swift-test/main.swift

```diff
@@ -54,6 +54,9 @@ class importTest: XCTestCase {
     XCTAssertNil(FirebaseApp.app())
     XCTAssertEqual(GULAppEnvironmentUtil.deviceModel(), "x86_64")
 
+    let majorVersion = try XCTUnwrap(FirebaseVersion.version().first)
```

Comment:
I'm a little confused as what this `first` call is doing here - the `version` is a non-optional String. Perhaps doing something like:

- Split by `.`
- Assert there are 3 items in the array.
- Assert each item in that array is a number
- Maybe assert the first number is 7? Seems like the benefit outweighs the cost of having to trivially update it next breaking change.